code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -110,7 +110,7 @@
 		return
 
 	if(isanimal(M))
-		if(M.size <= SIZE_TINY) //Fuck off mice
+		if(M.size <= SIZE_TINY && !istype(M, /mob/living/simple_animal/chicken)) //Fuck off mice
```

Comment:
Noooooo

Reply:
I blew it, didn't I?

Reply:
Changing only the size makes it so you can't buckle chickens to chairs and beds, kind of a big deal for chicken farmers. Adding the exception alone does nothing, since chickens would have have SIZE_SMALL. Both together should make chickens fit in bags and still be able to be buckled.

Reply:
A good way to code it then would be to have a bit flag.
I'll give you an example tommorow when I'm not half-asleep typing

Reply:
but the example never came